index.bs

```diff
@@ -6554,6 +6585,134 @@ requests will be affected.
 
 </div>
 
+#### The network.setCacheBehavior Command ####  {#command-network-setCacheBehavior}
+
+The <dfn export for=commands>network.setCacheBehavior</dfn> command configures
+the network cache behavior for certain requests.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+    <pre class="cddl remote-cddl">
+      network.SetCacheBehavior = (
+        method: "network.setCacheBehavior",
+        params: network.SetCacheBehaviorParameters
+      )
+
+      network.SetCacheBehaviorParameters = {
+        cacheBehavior: "default" / "bypass",
+        ? contexts: [+browsingContext.BrowsingContext]
+      }
+    </pre>
+   </dd>
+   <dt>Return Type</dt>
+   <dd>
+    <pre class="cddl">
+      EmptyResult
+    </pre>
+   </dd>
+</dl>
+
+<div algorithm>
+The <dfn export>WebDriver BiDi cache behavior</dfn> steps given |request| are:
+
+1. Let |context| be null.
+
+1. If |request|'s [=request/window=] is an [=environment settings object=]:
+
+  1. Let |environment settings| be |request|'s [=request/window=]
+
+  1. If there is a [=/browsing context=] whose [=active window=] is |environment
+     settings|' [=environment settings object/global object=], set |context| to
+     the [=top-level browsing context=] for that browsing context.
+
+1. If |context| is not null and [=navigable cache behavior map=] [=set/contains=]
+   |context|, return [=navigable cache behavior map=][|context|].
+
+1. Return [=default cache behavior=].
+
+</div>
+
+<div algorithm>
+The <dfn>context cache behavior</dfn> steps given |context| are:
+
+1. Set |top-level context| to the [=top-level browsing context=] for |context|.
+
+1. If [=navigable cache behavior map=] [=map/contains=] |top-level context|, return
+   [=navigable cache behavior map=][|top-level context|].
+
+1. Return [=default cache behavior=].
+
+</div>
+
+<div algorithm="remote end steps for network.setCacheBehavior">
+The [=remote end steps=] given <var ignore>session</var> and |command parameters| are:
+
+1. Let |behavior| be |command parameters|["<code>cacheBehavior</code>"].
+
+1. If |command parameters| does not [=map/contain=] "<code>contexts</code>":
+
+  1. Set the [=default cache behavior=] to |behavior|.
+
+  1. [=map/Clear=] [=navigable cache behavior map=].
+
+  1. Switch on the value of behavior:
+    <dl>
+      <dt>"<code>bypass</code>"
```

Comment:
@jgraham one more question regarding the PR of the Fetch specification. Would it be helpful to actually export the two cache modes so that we don't have to use string values?

Reply:
Maybe, although we still have to use a string, it just becomes a string we can link to something. Fetch itself doesn't seem to consistently do that e.g. it doesn't for cache-mode, but we could. In any case since that doesn't have any normative impact we can easily do it in a followup.

Reply:
Sounds fine with a follow-up if needed.